cves/kernel/CVE-2021-38203.yml

```diff
@@ -221,7 +222,7 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
+  name: ["fs", "btrfs"]
   note:
```

Comment:
could you clarify on what these stand for? these acronyms are a bit vague to figure out where this is really happening